lib/const.rb

```diff
@@ -102,17 +103,17 @@
 CREW_DEST_MAN_PREFIX  = File.join(CREW_DEST_DIR, CREW_MAN_PREFIX)
 
 # Local constants for contributors.
-CREW_LOCAL_REPO_ROOT = `git rev-parse --show-toplevel 2> /dev/null`.chomp
+CREW_LOCAL_REPO_ROOT = Dir.exist?('.git') ? `git rev-parse --show-toplevel`.chomp : nil
```

Comment:
Is there another way to execute beside the backticks?  Would a system call be more Ruby-like?

Reply:
The backticks do work...

Reply:
I know they work but the whole gist of this PR is to Rubyize and there may be an alternative "best practice" way to do it in Ruby.

Reply:
Would `exec` work?  See https://apidock.com/ruby/Kernel/exec.

Reply:
> Would `exec` work? See https://apidock.com/ruby/Kernel/exec.

No. Exec **Replaces the current process by running the given external _command_**.

See https://stackoverflow.com/a/18623297

I'm not sure it makes sense to fiddle with this. backticks do the job, and it's easy to see what's being invoked...

Reply:
Also:
![image](https://github.com/chromebrew/chromebrew/assets/1096701/2992f840-58e3-486e-b793-bab0560fc023)
from https://stackoverflow.com/a/31572431

Reply:
How about https://apidock.com/ruby/v2_5_5/Kernel/syscall?  That returns the output.

Reply:
> How about https://apidock.com/ruby/v2_5_5/Kernel/syscall? That returns the output.

The `syscall` function is for calling low-level kernel system calls like `read()` and `exec()`, not for external executables😅

We can use `Open3.capture2('cmd')[0]` or `%x[cmd]` to rubyize it, the former is better (it will not call `/bin/sh` if the command is passed in array) but longer